File: src/cxx/VulkanContext/Exceptions/NotSupportedLayerException.cpp
//
// Created by kgaft on 7/26/23.
//

#include "NotSupportedLayerException.h"

NotSupportedLayerException::NotSupportedLayerException(const char *message) : message(message) {

}

const char *NotSupportedLayerException::what() {
    return message;
}

File: src/main/java/com/kgaft/VulkanContext/Exceptions/NotSupportedLayerException.java
package com.kgaft.VulkanContext.Exceptions;

public class NotSupportedLayerException extends Throwable {
    public NotSupportedLayerException(String message) {
        this.message = message;
    }
    private String message;
    @Override
    public String getMessage() {
        return message;
    }
}
